salt/docs/config/nginx.docs-backend.conf

```diff
@@ -36,12 +36,111 @@ server {
     }
 
     # Smooth the switch between versions by mapping old files to their new location
-    location ~ ^/([a-z-]*/)?(3|3.5|3.6|3.7|3.8)/library/sets.html$ {
-        return 301 https://$host/$1$2/library/stdtypes.html#set-types-set-frozenset;
-    }
     location ~ ^/([a-z-]*/)?(3|3.6|3.7|3.8)/library/email.util.html$ {
         return 301 https://$host/$1$2/library/email.utils.html;
     }
+    location ~ ^/([a-z-]*/)?(3|3.5|3.6|3.7|3.8|3.9|3.10)/c-api/(class|cobject|int|string).html$ {
+        return 301 https://$host/$1$2/;
+    }
+    location ~ ^/([a-z-]*/)?(3|3.5|3.6|3.7|3.8|3.9|3.10)/howto/(doanddont|webservers).html$ {
+        return 301 https://$host/$1$2/;
```

Comment:
```suggestion
        return 301 https://$host/$1$2/howto/;
```

?

Reply:
I think if I clicked on https://docs.python.org/3/howto/webservers.html I don't care about other how-to's, I care to know that the link no longer works and I think linking to the homepage of the docs is better for that.

Reply:
right.